recipes/apr/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set name = "apr" %}
+{% set version = "1.5.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://archive.apache.org/dist/{{ name }}/{{ name }}-{{ version }}.tar.bz2
+  sha256: 7d03ed29c22a7152be45b8e50431063736df9e1daa1ddf93f6a547ba7a28f67a
+  patches:
+    # Skip a failing test on Mac due to `poll`'s behavior.
+    #
+    # ref: https://www.mail-archive.com/dev@apr.apache.org/msg26625.html
+    #
+    - macos_testpoll.c.patch  # [osx]
```

Comment:
Needed this patch to workaround Mac's `poll` (mis)behavior, which causes test failures. Please see the ref below or just google. 😒

ref: https://www.mail-archive.com/dev@apr.apache.org/msg26625.html